resources/aws_ec2.go

```diff
@@ -69,6 +73,10 @@ const (
 	SnsPolicyService = "events.amazonaws.com"
 	// SnsPolicyAction is the specific permission we are granting in the policy.
 	SnsPolicyAction = "SNS:Publish"
+	// SnsCertURLRegex is used to make sure we only download certificates
+	// from amazon. This regex will match "https://sns.***.amazonaws.com/"
```

Comment:
Either comment or regex is wrong since there is a dash (`-`) you didn't talk about.